.circleci/config.yml

```
@@ -28,7 +28,7 @@ jobs:
             chmod 755 terraform-docs-v${TERRAFORM_DOCS_VERSION}-linux-amd64
             mv terraform-docs-v${TERRAFORM_DOCS_VERSION}-linux-amd64 ~/bin/terraform-docs
           environment:
-            TERRAFORM_DOCS_VERSION: "0.12.1"
+            TERRAFORM_DOCS_VERSION: "0.14.0"
```

```suggestion
            TERRAFORM_DOCS_VERSION: "0.14.1"
```

There is a new version out.

---

You're having an issue with circleci tests because terraform-docs updated the way their code releases are managed. Here is the updated code you need to fix it:

```sh
>>            curl -sSLO https://github.com/terraform-docs/terraform-docs/releases/download/v${TERRAFORM_DOCS_VERSION}/terraform-docs-v${TERRAFORM_DOCS_VERSION}-linux-amd64.tar.gz
>>            curl -sSLO https://github.com/terraform-docs/terraform-docs/releases/download/v${TERRAFORM_DOCS_VERSION}/terraform-docs-v${TERRAFORM_DOCS_VERSION}.sha256sum
>>            [ $(sha256sum terraform-docs-v${TERRAFORM_DOCS_VERSION}-linux-amd64.tar.gz | cut -f1 -d' ') = $(grep linux-amd64 terraform-docs-v${TERRAFORM_DOCS_VERSION}.sha256sum | cut -f1 -d' ') ]
              tar -xf terraform-docs-v${TERRAFORM_DOCS_VERSION}-linux-amd64.tar.gz
              mv terraform-docs ~/bin/terraform-docs
```

---

The first two commands run without a problem, but I'm getting the following error message at the third command. 

```
zsh: command not found: sha256sum
grep: terraform-docs-v.sha256sum: No such file or directory
tar: Error opening archive: Failed to open 'terraform-docs-v-linux-amd64.tar.gz'
mv: rename terraform-docs to /Users/redkayak/bin/terraform-docs: No such file or directory
```

I have a file names `terraform-docs` but it's in the following location: 
```
/usr/local/Cellar/terraform-docs/0.14.1/bin
```

---

@austinlasseter - don't forget to update this to:

```suggestion
            TERRAFORM_DOCS_VERSION: "0.14.1"
```